content/core-features/authentication.textile

```diff
@@ -211,6 +211,53 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, function(err, tokenDe
 
 *Note:* The authenticity of the JWT *will not* be checked, due to Ably not having access to your SECRET key.
 
+h4(#jwt-claims). Using JWT for authenticated user claims
+
+It is possible to use JWTs to authenticate claims for users in your channels, allowing or disallowing certain interactions.
+
+Messages can include trusted fields in the `extras` object that contain information copied from their authentication token, making it unique to each user. These fields are deemed trusted as their content is added is supplied directly from Ably and not the client. The trusted field is a channel-specific piece of information and needs to be set for other channels if similar permissions are desired.
```

Comment:
```suggestion
Messages can be annotated with trusted metadata copied from the client's authentication token by Ably servers. Clients are unable to directly publish messages with user claim metadata. Claims contained within the authentication token are signed to prevent tampering. Claims can be scoped to individual channels or to namespaces of channels. <Link to the capability resource specifiers>. The most specific user claim will be annotated to the message as part of the `extras` object.
```